js/zonda.js

```diff
@@ -578,40 +578,54 @@ module.exports = class zonda extends Exchange {
 
     parseTicker (ticker, market = undefined) {
         //
-        //     {
-        //         m: 'ETH-PLN',
-        //         h: '13485.13',
-        //         l: '13100.01',
-        //         v: '126.10710939',
-        //         r24h: '13332.72'
-        //       }
+        //    {
+        //        market: {
+        //            code: 'ADA-USDT',
+        //            first: {
+        //                currency: 'ADA',
+        //                minOffer: '0.2',
+        //                scale: '6'
+        //            },
+        //            second: {
+        //                currency: 'USDT',
+        //                minOffer: '0.099',
+        //                scale: '6'
+        //            },
+        //            amountPrecision: '6',
+        //            pricePrecision: '6',
+        //            ratePrecision: '6'
+        //        },
+        //        time: '1655812661202',
+        //        highestBid: '0.492',
+        //        lowestAsk: '0.499389',
+        //        rate: '0.50588',
+        //        previousRate: '0.504981'
+        //    }
         //
-        const open = this.safeString (ticker, 'r24h');
-        const high = this.safeString (ticker, 'h');
-        const low = this.safeString (ticker, 'l');
-        const volume = this.safeString (ticker, 'v');
-        const marketId = this.safeString (ticker, 'm');
-        market = this.safeMarket (marketId, market, '-');
-        const symbol = market['symbol'];
+        const tickerMarket = this.safeValue (ticker, 'market', {});
+        const marketId = this.safeString (tickerMarket, 'code');
+        market = this.safeMarket (marketId, market);
+        const timestamp = this.safeInteger (ticker, 'time');
+        const rate = this.safeValue (ticker, 'rate');
         return this.safeTicker ({
-            'symbol': symbol,
-            'timestamp': undefined,
-            'datetime': undefined,
-            'high': high,
-            'low': low,
-            'bid': undefined,
+            'symbol': this.safeSymbol (marketId, market),
+            'timestamp': timestamp,
+            'datetime': this.iso8601 (timestamp),
+            'high': undefined,
+            'low': undefined,
+            'bid': this.safeNumber (ticker, 'highestBid'),
```

Comment:
We should pass string amounts, prices, costs and fees to safeTicker:

```suggestion
            'bid': this.safeString (ticker, 'highestBid'),
```